ethereum/core/src/merkle/utils/mod.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Argument Computer Corporation
 // SPDX-License-Identifier: Apache-2.0
 
+use std::mem::size_of;
```

Comment:
This fixes a compile error when building with the current version of the succinct toolchain (v1.79):
```
    |
121 |     let usize_len = size_of::<usize>();
    |                     ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
4   + use core::mem::size_of;
    |
4   + use std::mem::size_of;
    |
```